Aplib.Core/Belief/Beliefs/Belief.cs

```diff
@@ -1,6 +1,4 @@
-﻿using System;
```

Comment:
So why did you remove this everywhere? I'm not too much of a fan of it

Reply:
It causes a naming collision between `System.Action` and `Aplib.Core.Intent.Actions.Action`. This has already caused som confusion amongst ourselves when writing code, because you might think you are creating or using an aplib Action but you are actually using a System.Action. This can also created very confusing error messages.